### PR TITLE
Adde new variable group to all pipelines

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Add new variable group to pipelines

### CODE REVIEW

File: eng/pipelines/pipeline-generation.yml [hunk 1]
```diff
@@ -16,40 +16,40 @@ jobs:
         Branch: master
         Prefix: java
         PublicOptions: ''
-        InternalOptions: '--variablegroup 1 --variablegroup 58 --variablegroup 76 --variablegroup 56'
+        InternalOptions: '--variablegroup 1 --variablegroup 58 --variablegroup 76 --variablegroup 56 --variablegroup 93'
```
Comment (scbedd) [inline eng/pipelines/pipeline-generation.yml:19]: Supports APIView?

**Review by scbedd — APPROVED** [review]
Asked which group this is, but won't protest if it doesn't seem outlandish.

File: eng/pipelines/pipeline-generation.yml (hunk 1 above)
Comment (praveenkuttappan) [reply eng/pipelines/pipeline-generation.yml:19]: Yes. New variable group to support APIView tool

**Comment by praveenkuttappan** [comment]
@scbedd : This is new variable group with APIKey and apiView upload URL details. Name: APIReview-AutoCreate Configurations

**Pull request merged**